content/2020-06-05-pycones-2020-c4p-en.md

```diff
@@ -0,0 +1,101 @@
+
+<center><small>
+NOTE: Si hablas español, puedes [ir a la versión es ESPAÑOL]({filename}/2020-06-05-pycones-2020-c4p-es.md)
+</small></center>
+
+We are pleased to announce that the C4P for **PyConES 2020** *(Online version)* is open!
+
+Now you can send us your proposals for the PyConES 2020 conference and this year we will have some important changes!
+
+
+## Our Code of Conduct
+
+We want to have a welcoming and pleasant conference, with a harassment-free experience for everyone. PyConES 2020 has a mandatory [code of conduct](https://www.es.python.org/pages/codigo-de-conducta.html) for all people participating in the event.
+
+
+## Where and when:
+
+The eighth edition of PyConES will be held on **October 3rd, 2020** as an online event.
+
+
+## What is the deadline to submit my proposal:
+
+You can submit all your proposals until 23:59 on **Sunday, July 26, 2020**.
+
+
+## Type of sessions available
+
+As per the current situation this year we’re changing the format and limiting it to just one option, the 20 minutes short talk, in order to make it easier for both speakers and attendees.
+
+So, you’ll have a 20 minutes slot divided in 15 minutes for the talk and 5 minutes for questions from the attendees..
+
+
+## What can I talk about:
+
+As this year there just be a single track of talks and given that there are specific events for Data Science and ML, we decided to focus on topics that have less opportunities to be shown and that are also relevant for the Python community. We’d like to propose the following topics:
+
+* **Core Python** - Language Features, Python Implementations, Extending Python and Standard Library, language internals.
+
+* **Community, culture and society** - Diversity, health, productivity, workspace issues, privacy, community building, coding for causes.
+
+* **Web** -  Web development, APIs, Microservices...
+
+* **Desktop Applications** - Qt, GTK+, Tkinter, Gnome, KDE, Accessibility.
+
+* **Game, Design and 3D Modelling **- Python in developing games, 3-D modelling and animation.
+
+* **Embedded Python and IoT** - MicroPython, Python on Hardware, Robotics, Arduino and Raspberry Pi.
+
+* **Sys / Dev Ops** - Testing, CI/CD, Containers, Orchestration, Logging and Monitoring.
+
+* **Networking and Security** - Network Programming, Network Security and Encryption
+
+At PyConES we have an audience with different backgrounds and diverse interests. Therefore, we accept proposals that offer an introduction for those newly initiated,  as well as those proposals that deepen in some aspect in particular for those who dominate the topic.
+
+
+## How will the selection process be?
+
+1. You have to send us a link to a video of no more than 5 minutes with the talk’s summary. Please note that if you’re selected we’ll ask you to record a 15’ video that will be broadcasted during the event.
```

Comment:
Perhaps include a link to the form submit the proposal video from this paragraph, so people don't mistakenly use the contact@... mail to do that.